### PR TITLE
add terminal cli  :cl: 

### PR DESCRIPTION
**Pull request by juliomatcom**
add an initial terminal cli for this app

### CODE REVIEW

**Comment by alvaropinot** [comment]
where are those unit tests? 👅  hehehe

File: bin/deprikat [hunk 1]
```diff
@@ -0,0 +1,41 @@
+'use strict';
```
Comment (alvaropinot) [inline bin/deprikat]: what about a simpler cli something like

```
#!/usr/bin/env node
require('./cli.js');
```

then inside `cli.js` all this file?

indeed I think that bash info is quite important :P

**Comment by juliomatcom** [comment]
added commander  https://github.com/alvaropinot/deprikat/pull/15/commits/c506b19627b92f2236e0b768fee74adfc08db9c8 to improve the program, sorry for the bad commits messages :cry:

**Comment by juliomatcom** [comment]
im closing this PR and moving in favor of a new **branch**

**Pull request closed without merging**